src/js/claims-status/utils/appeals-v2-helpers.jsx

```diff
@@ -253,64 +253,64 @@ export function getStatusContents(statusType, details = {}, name = {}) {
-      contents.title = 'Your hearing has been scheduled';
+      contents.title = 'The Board has scheduled your hearing';
       contents.description = (
-        <p>Your {getHearingType(details.type)} hearing is scheduled for {formattedDate} at {details.location}. If you need to change this
-        date, please contact your Veteran Service Organization or representative as soon as
+        <p>The Board scheduled your {getHearingType(details.type)} hearing for {formattedDate} at {details.location}. If you need to change this
+        date, please contact your Veterans Service Organization or representative as soon as
         possible.</p>
       );
       break;
     }
     case STATUS_TYPES.onDocket:
       contents.title = 'Your appeal is waiting to be assigned to a judge';
       contents.description = (
-        <p>Your appeal is at the Board of Veterans’ Appeals waiting to be assigned to a Veterans
-        Law Judge. Staff at the Board will make sure that your case is complete, accurate, and
+        <p>Your appeal is at the Board of Veterans’ Appeals, waiting to be assigned to a Veterans
+        Law Judge. Staff at the Board will make sure your case is complete, accurate, and
         ready to be decided by a judge.</p>
       );
       break;
     case STATUS_TYPES.atVso:
-      contents.title = 'Your appeal is currently with your Veteran Service Organization';
+      contents.title = 'Your Veterans Service Organization is preparing your appeal';
       contents.description = (
-        <p>{details.vsoName} is currently preparing a document in support of your appeal. For more information,
-        please contact your Veteran Service Organization or representative.</p>
+        <p>{details.vsoName} is preparing a document in support of your appeal. For more information,
+        please contact your Veterans Service Organization or representative.</p>
       );
       break;
     case STATUS_TYPES.decisionInProgress:
       contents.title = 'A judge is reviewing your appeal';
       contents.description = (
         <p>Your appeal is at the Board of Veterans’ Appeals being reviewed by a Veterans Law Judge
         and their team of attorneys. If you submit evidence that isn’t already included in your
-        case, this may delay your appeal.</p>
+        case, it may delay your appeal.</p>
       );
       break;
     case STATUS_TYPES.bvaDevelopment:
-      contents.title = 'The judge is seeking additional information before making a decision';
+      contents.title = 'The judge is asking for more information before making a decision';
       contents.description = (
-        <p>The Board of Veterans’ Appeals is seeking evidence or an outside opinion from a legal,
-        medical, or other professional necessary to make decision about your appeal.</p>
+        <p>The judge reviewing your case at the Board of Veterans’ Appeals is asking for more evidence or an outside opinion from a legal,
```

Comment:
revert back to 
<p>The Board of Veterans’ Appeals is seeking evidence or an outside opinion from a legal, medical, or other professional necessary to make decision about your appeal.</p>

Reply:
Thanks @gnakm, updated.